dataset-spec.html

```diff
@@ -242,13 +224,31 @@ <h3>Methods</h3>
     </p>
     <p>Returns a new dataset containing all quads returned by applying <code>iteratee</code> to each quad in the current dataset.</p>
 
+    <p>
+      <dfn>reduce</dfn>
+    </p>
+    <p>
+      This method calls the <code>iteratee</code> for each <code>quad</code> of the <a>DatasetCore</a>.
+      The first time the <code>iteratee</code> is called, the <code>accumulator</code> value is the <code>initialValue</code> or, if not given, equals to the first quad of the Dataset.
+      The return value of the <code>iteratee</code> is used as <code>accumulator</code> value for the next calls.
+    </p>
+    <p>This method returns the return value of the last <code>iteratee</code> call.</p>
+    <p>Note: This method is aligned with Array.prototype.reduce() in ECMAScript-262.</p>
+
     <p>
       <dfn>some</dfn>
     </p>
     <p>Existential quantification method, tests whether some quads in the dataset pass the test implemented by the provided <code>iteratee</code>.</p>
     <p>This method immediately returns boolean true once a quad that passes the test is found.</p>
     <p>Note: this method is aligned with Array.prototype.some() in ECMAScript-262.</p>
 
+    <p>
+      <dfn>toArray</dfn>
+    </p>
+    <p>Returns the set of quads within the dataset as a host language native sequence, for example an Array in ECMAScript-262.</p>
+    <p>Note: a sequence in [WEBIDL] is passed by value, not by reference.</p>
```

Comment:
I don't know the intent of this note. IMO we should either remove it or clarify why this is noteworthy. (The way I read it is: *be careful, a sequence is supposed to be passed by value*, which doesn't make much sense since in JavaScript everything is passed by value.)

Reply:
It was copied from the [RDF Interfaces spec](https://www.w3.org/TR/rdf-interfaces/). But you are right. I think it should be removed. But this PR should be just about moving stuff. So let's do these small cleanup stuff in another PR.